src/defineLayout/createNestControls/enforceNestLogic.js

```diff
@@ -0,0 +1,17 @@
+export default function enforceNestLogic(id_cols) {
+    // limit select options to those that are not already selected (except for None - want to keep that around)
+    this.containers.nestControls
+        .selectAll('select')
+        .selectAll('option')
+        .style('display', d => (id_cols.includes(d.value_col) ? 'none' : null));
```

Comment:
only concern here is that it's not immediately apparent that the second nest has to be deselected for the first nest to include the second nest option.  would it be overdoing it to set the second nest to none when the first nesting matches the second nesting, or to the third nesting if selected?

Reply:
good point - that's probably more natural

Reply:
made [an issue](https://github.com/RhoInc/crf-heat-map/issues/128), your call on which route works better.